src/app/control-planes/views/ControlPlaneDetailView.vue

```diff
@@ -13,109 +13,164 @@
         </h1>
       </template>
 
-      <div
-        class="stack"
-        data-testid="detail-view-details"
+      <DataSource
+        v-slot="{ data: globalInsight, error: globalInsightError }: GlobalInsightSource"
+        src="/global-insight"
       >
-        <ControlPlaneStatus />
-
-        <div class="columns">
-          <KCard v-if="can('use zones')">
-            <template #body>
-              <DataSource
-                v-slot="{ data, error }: ZoneOverviewCollectionSource"
-                src="/zone-cps?page=1&size=10"
+        <ErrorBlock
+          v-if="globalInsightError"
+          :error="globalInsightError"
+        />
+
+        <LoadingBlock v-else-if="globalInsight === undefined" />
+
+        <template v-else>
+          <div
+            class="stack"
+            data-testid="detail-view-details"
+          >
+            <template v-if="globalInsight.meshes.total <= 1 && globalInsight.dataplanes.standard.total === 0">
```

Comment:
**Question**: Does this condition make sense?

We previously checked for `meshes.total === 1`, but I suppose the onboarding alert is equally is useful when there aren’t any meshes, right?

Also, this is missing the check for whether the one mesh was the default mesh.

Reply:
tbh I'm not sure of the reasoning behind not doing this anymore, it no longer costs us an additional HTTP request anymore, in fact the OnboardingNotification kinda no longer makes HTTP requests anymore (depending on the race for sharing of course).

Is there a case that we want to show the onboarding notification if you have deleted (or otherwise don't have) the `default` mesh, and have possibly replaced it with another one? Maybe we said that in the sync we had, but if we did could you remind me as I don't remember.

Maybe this is related to:

> but I suppose the onboarding alert is equally is useful when there aren’t any meshes, right?

Wouldn't zero meshes mean this isn't a fresh install anymore, and therefore don't show the onboarding? Is there a case where zero meshes means "fresh install"?

Reply:
I changed this to check for the default mesh again and to only trigger when there is exactly one mesh.